webclient/src/components/views/Create.vue

```diff
@@ -1,13 +1,16 @@
 <template>
   <div>
     <LoggedInHeader />
-    <div class='create-container'>
+    <div class='create-container' v-if="loading">
```

Comment:
I think the logic on this and the one below is flipped?

Reply:
Whoops, yes that is correct. This is flipped.